added newsletter section, moved about pydata and numfocus section down on the page, removed conf info from main site (#56)
PyDataAmsterdam/amsterdam.pydata.org: _posts/newsletters/2018-05-25-pydata-amsterdam-2018.md

---
layout: post
title: Newsletter 2020-04-21
category: [newsletter]
---

Due to the unfortunate circumstances surrounding COVID-19, we can not come together in person. That doesn't mean though that PyData Amsterdam is done. Over the years we've been building up a fantastic repertoire of recordings at both PyData Amsterdam and at other PyData events across the world. In the coming period, we'll highlight some of our favourites for you to enjoy as well as keep you updated with some of our and the rest of the communities’ plans.

## Our picks:

### The evolution of array computing in Python - Ralf Gommers (PyData Amsterdam 2019)

> https://www.youtube.com/watch?v=HVLPJnvInzM

Reviewer: Also here.

```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/HVLPJnvInzM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Have you ever wanted to run your NumPy based code on multiple cores, or on a distributed system, or on your GPU? Wouldn't it be nice to do this without changing your code? In this talk, Ralf describes the addition of two protocols to numpy which attempt to provide a bridge between those different array implementations

Ralf has been deeply involved in the SciPy and PyData communities for over a decade. He is a core developer of NumPy, SciPy and PyWavelets, and has contributed widely throughout the SciPy ecosystem.

### Let them write code - Ines Montani (PyCon India 2019)

> https://www.youtube.com/watch?v=Ivb4AAuj5JY

Reviewer: Shall we make all youtube links into embeds? It seems to work nicely even though Markdown doesn't officially support it.

```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/Ivb4AAuj5JY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Ines is a co-founder of Explosion, known for developing of spaCy, a leading open-source NLP library. In  “Let Them Write Code”, she shares Explosions philosophy for building developer tools, and why spaCy’s user-facing API works the way it does. This talk is not only useful when you build developer-focused tools, but will also help you structure your own internal codebases.

### Seasonality modelling from scratch - Vadim Nelidov (PyData Amsterdam Code Breakfast)

> https://godatadriven.com/topic/recording-online-code-breakfast-seasonality-modelling-from-scratch/ password: Breakfast

Reviewer: Linkify?
```suggestion
[Seasonality modelling from scratch](https://godatadriven.com/topic/recording-online-code-breakfast-seasonality-modelling-from-scratch/) password: Breakfast
```

In case you missed it, this is the recording of our previous virtual PyData Amsterdam Code Breakfast. In this session Vadim explained how time series are built up,how to decompose them into seasonal components and some of the dangers with naive approaches to detecting seasonality.

A new code breakfast:  Chatbots from the Command Line
It’s once again code breakfast time! We meet online, but we’ll keep the positive vibe kept share some breakfast recipes! You’ll get to build your first chatbot from the command line using the open source tool Rasa open source and python. The breakfast will be hosted by Vincent Warmerdam, Research advocate at Rasa.

> We'll also briefly discuss topics in NLP in general, like named entity recognition but the main goal is to get a working assistant running locally. Rasa works on top of Tensorflow 2 but you won't need to be familiar with it to get it to run. If you want to join, please sign up here: https://www.meetup.com/nl-NL/PyData-NL/events/270156567/

Reviewer: ```suggestion
We'll also briefly discuss topics in NLP in general, like named entity recognition but the main goal is to get a working assistant running locally. Rasa works on top of Tensorflow 2 but you won't need to be familiar with it to get it to run. If you want to join, please sign up [at the meetup page](https://www.meetup.com/nl-NL/PyData-NL/events/270156567/)
```

## Other news:

> The Python Pizza Conference is happening this weekend, on April 25th. It’s a fully virtual conference featuring several talks by old friends of PyData Amsterdam like Alexander Hendorf, Vincent Warmerdam, Cheuk Ting Ho and Ian Ozsvald. Tickets are €30  and all proceeds go to Doctors Without Borders. For more info, visit https://remote.python.pizza/.

Reviewer: ```suggestion
The Python Pizza Conference is happening this weekend, on April 25th. It’s a fully virtual conference featuring several talks by old friends of PyData Amsterdam like Alexander Hendorf, Vincent Warmerdam, Cheuk Ting Ho and Ian Ozsvald. Tickets are €30  and all proceeds go to Doctors Without Borders. For more info, visit [remote.python.pizza](https://remote.python.pizza/).
```
